docs/instrumenten/DPIA.md

```diff
@@ -0,0 +1,58 @@
+vereiste:
+- beperkte_bewaartermijn_van_persoonsgegevens
+- beschrijven_en_toewijzen_van_verantwoordelijkheden_bij_verwerking_persoonsgegevens
+- dpia_verplicht_bij_hoog_risico
+- inroepen_privacyrecht_bij_verwerking_persoonsgegevens
+- juistheid_en_actualiteit_van_persoonsgegevens
+- persoonsgegevens_worden_rechtmatig_verwerkt
+- privacy_bij_ontwerp_bij_verwerking_van_persoonsgegevens
+- transparantie_bij_verwerken_persoonsgegevens
+- verdere_verwerking_van_persoonsgegevens_in_ai_testomgevingen
+levenscyclus:
+- probleemanalyse
+- ontwerp
+- dataverkenning-en-datapreparatie
+bouwblok:
+- privacy-en-gegevensbescherming 
+rollen:
+- ethicus
+- jurist 
+- privacy-officer
+---
+
+<!-- tags -->
+
+[Direct naar de DPIA](https://www.autoriteitpersoonsgegevens.nl/themas/basis-avg/praktisch-avg/data-protection-impact-assessment-dpia){ .md-button }
+## Instrument
+
+Is uw organisatie van plan persoonsgegevens te verwerken, maar levert dat waarschijnlijk een hoog privacyrisico op? Dan is de organisatie verplicht eerst een 'data protection impact assessment' (DPIA) uit te voeren.Als organisatie moet u zelf bepalen of uw gegevensverwerking een hoog privacyrisico oplevert. En u dus een DPIA moet uitvoeren. De volgende criteria kunnen u hierbij helpen:
+
+*  Wat er in de Algemene verordening gegevensbescherming (AVG) staat over wanneer u een DPIA moet uitvoeren.
+*  De lijst van de Autoriteit Persoonsgegevens (AP) met soorten verwerkingen waarvoor u een DPIA moet uitvoeren.
+*  De 9 criteria voor een DPIA van de Europese privacytoezichthouders.
+
+De AVG geeft aan dat u in ieder geval een DPIA moet uitvoeren als u als organisatie:
+
+* Systematisch en uitgebreid persoonlijke aspecten van mensen beoordeelt. Dit doet u op basis van geautomatiseerde verwerking van persoonsgegevens, waaronder profiling.
+* En hierop baseert u besluiten die gevolgen hebben voor mensen. Bijvoorbeeld dat zij geen lening kunnen afsluiten. Een voorbeeld hiervan is creditscoring.
+* Op grote schaal bijzondere persoonsgegevens verwerkt.
+* Strafrechtelijke gegevens verwerkt.
+* Op grote schaal en systematisch mensen volgt in een publiek toegankelijk gebied. Bijvoorbeeld met cameratoezicht.
+
+Een DPIA moet in een vroeg stadium van de beleids- of projectontwikkeling worden uitgevoerd. Op dat moment kan namelijk nog zonder vooroordelen worden nagedacht over de gevolgen en kan het voorstel nog makkelijker worden herzien. Dit voorkomt ook latere, kostbare aanpassingen in processen, herontwerp van systemen of zelfs stopzetten van een project. Behalve aan het begin van een project kun  een DPIA ook op andere momenten en meermaals worden uitgevoerd en geactualiseerd. Als het voorstel wijzigt, wordt een DPIA opnieuw uitgevoerd. Als de gegevensverwerkingen of de gevolgen ervan veranderen, moet de DPIA worden geactualiseerd. Volgens de European Data Protection Board (EDPB) moet een DPIA iedere drie jaar worden geëvalueerd.
```

Comment:
```suggestion
Een DPIA moet in een vroeg stadium van de beleids- of projectontwikkeling worden uitgevoerd. 
Op dat moment kan namelijk nog zonder vooroordelen worden nagedacht over de gevolgen en kan het voorstel nog makkelijker worden herzien. 
Dit voorkomt ook latere, kostbare aanpassingen in processen, herontwerp van systemen of zelfs stopzetten van een project. 
Behalve aan het begin van een project kan een DPIA ook op andere momenten en meermaals worden uitgevoerd en geactualiseerd. 
Als het voorstel wijzigt, wordt een DPIA opnieuw uitgevoerd. 
Als de gegevensverwerkingen of de gevolgen ervan veranderen, moet de DPIA worden geactualiseerd. 
Volgens de European Data Protection Board (EDPB) moet een DPIA iedere drie jaar worden geëvalueerd.
```

Reply:
Extra enters toegevoegd. Het makkelijkst is om zoveel mogelijk in aparte regels te zetten, dat voorkomt conflicten